tests/Instances/Data/ShortText.hs

```diff
@@ -0,0 +1,22 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+{-|
+Module:      Instances.Data.ShortText
+Copyright:   (C) 2014-2018 Ryan Scott
+License:     BSD-style (see the file LICENSE)
+Maintainer:  Ryan Scott
+Stability:   Provisional
+Portability: GHC
+
+Provides an 'Arbitrary' instance for 'ShortText' values.
+
+-}
+
+module Instances.Data.ShortText () where
+
+import           Data.Text.Short (ShortText, fromString)
+
+import           Test.QuickCheck (Arbitrary (..))
+
+instance Arbitrary ShortText where
+    arbitrary = fromString <$> arbitrary
```

Comment:
The GHC 7.8 build is failing since `(<$>)` wasn't exported by the `Prelude` in that release. It's pretty easy to fix, though—just add the following imports:

```haskell
import Prelude ()
import Prelude.Compat
```